.github/workflows/playwright.yml

```diff
@@ -0,0 +1,30 @@
+name: Playwright Tests
+on:
+  deployment_status:
+jobs:
+  test:
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success'
+    # && !contains(github.event.deployment_status.target_url, 'guild-xyz-storybook')
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Debug
+        run: echo "${{ github.event.deployment_status.target_url }}"
+      - name: Install dependencies
+        run: npm ci
```

Comment:
I'm wondering if we need to install all of the dependencies 🤔 Later on this might be worth an investigation